src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsRecoveryObject.java

```diff
@@ -78,6 +78,11 @@ public void verifyPageFairRecoveryWithAdBlock() {
 
   public void verifyPageFairRecoveryWithNoAdBlock() {
     List<WebElement> markedSlots = driver.findElements(By.cssSelector("[adonis-marker]"));
+    Assertion.assertEquals(markedSlots.size(), RECOVERABLE_ADS_COUNT);
+  }
+
+  public void verifyPageFairRecoveryLoggedIn() {
```

Comment:
please, rename it to something more descriptive like verify-if-there-is-no-PageFair-recovery-elements-on-page or simillar :)